pkg/http-server/remote-repo.go

```diff
@@ -112,7 +112,7 @@ func (s *scanRemoteRepoReq) ScanRemoteRepo(iacType, iacVersion string, cloudType
 
 	// create a new runtime executor for scanning the remote repo
 	executor, err := runtime.NewExecutor(iacType, iacVersion, cloudType,
-		"", iacDirPath, "", policyPath)
+		"", iacDirPath, "", policyPath, []string{}, []string{})
```

Comment:
We should be taking rules to scan and skip for the API server as well. This functionality applies to the scanning for API server as well.

Reply:
Implemented